tests/beaker_tests/cisco_interface/test_interface.rb

```diff
@@ -364,37 +364,28 @@
   },
 }
 
-tests['negotiate'] = {
-  desc:               '5.1 negotiate-auto',
+tests[:auto] = {
+  desc:               '5.1 Misc. Auto Value Properties',
   operating_system:   'nexus',
-  platform:           'n(5|6|7)k',
   intf_type:          'ethernet',
-  preclean:           true,
   sys_def_switchport: false,
+  resource_name:      'cisco_interface',
```

Comment:
I was pretty sure I had a good reason, but testing has proven me wrong :-)

The `resource_name` changes will be removed in the next push.